master/getting-started/kubernetes/upgrade/delete.md

```diff
@@ -37,29 +37,7 @@ This procedure requires etcdctl v3. The etcdctl tool is installed along with etc
 > for more information about the flags and environment variables.
 {: .alert .alert-info}
 
-1. Issue the following command to retrieve a list of all of the Calico keys.
-   
-   ```
-   etcdctl --endpoint=<etcdv2-hostname:port> ls /calico --recursive
-   ```
-   
-1. Issue the following command to delete the {{site.prodname}} keys.
-   
-   ```
-   etcdctl --endpoint=<etcdv2-hostname:port> rm /calico/ --recursive 
-   ```
-   
-1. Issue the following command to confirm that the {{site.prodname}} keys were deleted.
-   
-   ```
-   etcdctl --endpoint=<etcdv2-hostname:port> ls /calico --recursive
-   ```
-   
-   It should return `Error: 100: Key not found (/calico) [1186]`.
-   
-1. Congratulations! You've cleaned {{site.prodname}}'s etcdv2 datastore of {{site.prodname}}
-   data. 
-
+{% include {{page.version}}/deleting-etcd-v2-data.md %}
```

Comment:
I notice that we are linking to the new include in the Kubernetes page but not from the OpenStack page.

Reply:
Are you sure?
```
~/go/src/github.com/projectcalico/calico/master/getting-started/openstack/upgrade$ grep "% include" delete.md 
{% include {{page.version}}/deleting-etcd-v2-data.md %}
{% include {{page.version}}/deleting-etcd-v3-data.md %}
```